src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderServiceTest.java

```diff
@@ -68,44 +67,27 @@ public void givenDeltaXmlInSftp_shouldBeSavedIntoCcd() throws IOException {
             anyString(),
             anyString(),
             anyString(),
-            anyString()))
-            .willReturn(StartEventResponse.builder().build());
+            anyString())
+        ).willReturn(StartEventResponse.builder().build());
 
-        given(idamApiClient.authorize(anyString()))
-            .willReturn(new Authorize("url", "accessToken"));
-
-        caseLoaderService.process();
-
-        verify(coreCaseDataService).startEventAndSaveGivenCase(any(CaseData.class));
-    }
-
-    @Test
-    @Ignore
-    // FIXME: 07/02/2018 Unignore this test when Reference data gets converted to a case
```

Comment:
Did you mean to delete this ignored test? This was put in as a placeholder so we don't forget to put an integration test in when Reference data gets converted

Reply:
I know we talked about it, but then I thought we can safely clean that up, remove it completely since we will have to implement the reference data denormalization work later on and for that work, we will have to add unit, integration and e2e tests anyway.